Refureku/Library/Include/Internal/Refureku/TypeInfo/Variables/VariableBaseImpl.inl

```diff
@@ -5,13 +5,18 @@
 *	See the LICENSE.md file for full license details.
 */
 
-inline VariableBase::VariableBaseImpl::VariableBaseImpl(char const* name, std::size_t id, EEntityKind kind, Type const& type, Entity const* outerEntity) noexcept:
+inline VariableBase::VariableBaseImpl::VariableBaseImpl(char const* name, std::size_t id, EEntityKind kind, char const* canonicalTypeName, Type const& type, Entity const* outerEntity) noexcept:
 	EntityImpl(name, id, kind, outerEntity),
-	_type{type}
+	_type{type}, _canonicalTypeName{canonicalTypeName}
 {
 }
 
 inline Type const& VariableBase::VariableBaseImpl::getType() const noexcept
 {
 	return _type;
+}
+
+inline const char* VariableBase::VariableBaseImpl::getCanonicalTypeName() const noexcept
```

Comment:
The project uses east const nomenclature so change to `char const*` for consistency.